src/app/view/components/listing-data-table/action-header/action-header.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('ActionHeaderComponent', () => {
             ],
             providers: [
                 { provide: MessageService, useValue: messageServiceMock },
-                ConfirmationService
+                DotConfirmationService
```

Comment:
we need test when this DotConfirmationService is used

Reply:
Nah... in fact, we need to do a refactor on how actions are working in ```action-header.component```